src/player.cpp

```diff
@@ -11410,20 +11410,20 @@ hint_rating player::rate_action_read( const item &it ) const
     return HINT_GOOD;
 }
 
-void player::read(int inventory_position)
+int player::read( int inventory_position, int inv_from )
```

Comment:
Don't cast it to int. If you're returning an enum, the return type should be said enum.